src/Atmos/Model/linear.jl

```diff
@@ -388,3 +388,119 @@ function numerical_flux_first_order!(
         direction,
     )
 end
+function numerical_flux_first_order!(
+    numerical_flux::RoeNumericalFluxMoist,
+    balance_law::AtmosLinearModel,
+    fluxᵀn::Vars{S},
+    normal_vector::SVector,
+    state_prognostic⁻::Vars{S},
+    state_auxiliary⁻::Vars{A},
+    state_prognostic⁺::Vars{S},
+    state_auxiliary⁺::Vars{A},
+    t,
+    direction,
+) where {S, A}
+
+    numerical_flux_first_order!(
+        CentralNumericalFluxFirstOrder(),
+        balance_law,
+        fluxᵀn,
+        normal_vector,
+        state_prognostic⁻,
+        state_auxiliary⁻,
+        state_prognostic⁺,
+        state_auxiliary⁺,
+        t,
+        direction,
+    )
```

Comment:
Is this a linearized flux ? why is this flux component nonlinear ? @mwarusz  This is also in the Dry roeflux

Reply:
I believe this uses the linearized pressure, can you point out which component is nonlinear?